### PR TITLE
OSDOCS-9847: adds 4.15.6 relnotes MicroShift

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.15

Issue:
[OSDOCS-9847](https://issues.redhat.com/browse/OSDOCS-9847)

Link to docs preview:
[4.15.6 async release note](https://73934--ocpdocs-pr.netlify.app/microshift/latest/microshift_release_notes/microshift-4-15-release-notes#microshift-4-15-6-dp)

QE review:
- N/A stock text, no bugs or enhancements

### CODE REVIEW

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

**Comment by mletalie** [comment]
Hello @ShaunaDiaz,
   Small comment above, and also going to assume like the other RN PR I did this AM, those links arent live yet, hence the 404? 
Thanks, LGTM!

File: microshift_release_notes/microshift-4-15-release-notes.adoc [hunk 1]
```diff
@@ -251,4 +251,13 @@ For the latest images included with {microshift-short}, view the contents of the
 ==== Enhancements
 
 Change in openshift-marketplace namespace security::
-* With this release, the `openshift-marketplace` namespace security defaults to `baseline`. See the OLM documentation for specifics on how this change impacts your operator deployments. See xref:../microshift_running_apps/microshift-operators-olm.adoc#microshift-operators-olm[Using Operator Lifecycle Manager with {microshift-short}]. (link:https://issues.redhat.com/browse/OCPBUGS-30034[OCPBUGS-30034])
\ No newline at end of file
+* With this release, the `openshift-marketplace` namespace security defaults to `baseline`. See the OLM documentation for specifics on how this change impacts your operator deployments. See xref:../microshift_running_apps/microshift-operators-olm.adoc#microshift-operators-olm[Using Operator Lifecycle Manager with {microshift-short}]. (link:https://issues.redhat.com/browse/OCPBUGS-30034[OCPBUGS-30034])
```
[the remark this answers is not in the document]
Comment (ShaunaDiaz) [reply microshift_release_notes/microshift-4-15-release-notes.adoc:254]: That actually didn't change with this PR. It's part of the previous release note. Got scooped into the diff for a reason I can't see.

**Pull request merged**